patroni/api.py

```diff
@@ -109,7 +109,7 @@ def do_GET(self, write_status_code_only=False):
         elif 'master' in path or 'leader' in path or 'primary' in path or 'read-write' in path:
             status_code = primary_status_code
         elif 'replica' in path:
-            status_code = replica_status_code
+            status_code = 200 if response.get('state') == 'running' else 503
```

Comment:
Your change fixes the problem only partially and unfortunately not 100% correctly.
Instead you should look on the line [104](https://github.com/zalando/patroni/pull/1152/files#diff-790530624eaa21e5fb614fc321aef277R104) and fix it.

Reply:
ok, got it, my bad. changes are commited